src/app/app.component.html

```diff
@@ -1,7 +1,7 @@
 <mat-toolbar color="primary">
   <ul class="navbar">
     <li>
-      <a href="/" class="page-logo">
+      <a href="https://gochain.io" target="_blank" class="page-logo">
```

Comment:
Does this have to be hard-coded? It would be nice if it functioned locally during development and wasn't fixed to the live url.

Reply:
I'll see if I can setup a config component.